tests/integration/utils.py

```diff
@@ -164,6 +144,39 @@ def _get_authentication_params(self):
         return {"auth": None}
 
 
+def download_archive(download_url, file_name_tar):
+    """
+    Download an archive.
+
+    :param download_url: URL to get the archive
+    :param file_name_tar: Path to the downloaded bundle
+    :return: Status of downloading
+    :raises requests.exceptions.HTTPError: if the request to the Cachito API fails
+    """
+    with requests.get(download_url, stream=True) as resp:
+        resp.raise_for_status()
```

Comment:
Since we are already checking the status code right after downloading the archive, should we still have this here?

Reply:
I would prefer to keep raising but delete additional checking out of function.

Reply:
Sounds good. Note that in this case, all we would get when the test fail is that the request failed. We could get more context doing the other way around, but I'm fine with either way you decide to go for.